docs/source/contributing.rst

```diff
@@ -650,6 +643,20 @@ Objects
 
     """
 
+Documenting your contribution
+-----------------------------
+
+Staged docs for every new PR are built and accessible at ``soundata.github.io/preview/PR-<#PR_ID>`` in which ``<#PR_ID>`` is the pull request ID. 
```

Comment:
That's the old URL :) new ones look like this: https://soundata--170.org.readthedocs.build/en/170/

Reply:
bon catch :)